registro.html

```diff
@@ -107,11 +107,10 @@ <h3 class="center-align tituloParrafo">Regístrate en PinWin</h3>
                     </div>
                 </div>
                 <div>
-                    <div class="row center" onmouseover = ChangeTextButton("ButtonRegistro")>
-                        <button id="ButtonRegistro" class="btn-large waves-effect waves-light red lighten-2 pulse" type="submit"
-                                name="action">
+                    <div class="row center">
+                        <button id="ButtonRegistro" class="btn-large  red lighten-2 pulse" type="submit"
```

Comment:
```suggestion
                        <button id="ButtonRegistro" class="btn-large  red lighten-2" type="submit"
```
Habría que quitarle ``pulse`` por defecto.